src/app/src/Map/Map.css

```diff
@@ -1,4 +1,10 @@
 .leaflet-container {
   width: 100vw;
   height: 100vh;
+}
+
+.button-container {
+  padding-left: 1rem;
+  padding-right: 1rem;
+  margin-top: 2rem;
```

Comment:
I played around with the styles locally and to get a style that didn't push the buttons down to a place the user couldn't scroll to since the map would just move, I got this to render like the wireframes: 
```css
.leaflet-container-wrapper {
  display: flex;
  flex-flow: column nowrap;
  height: 100vh;
}

.leaflet-container {
  width: 100vw;
  height: 100%;
  flex-grow: 1;
}

.button-container {
  display: flex;
  flex-flow: row nowrap;
  justify-content: center;
  z-index: 1000;
  width: 100vw;
  padding: 1rem 0;
  background-color: white;
}
```
Examples:
![image](https://user-images.githubusercontent.com/64238570/236639837-f588b34d-059d-41cf-ac2e-e8230c0d6ca2.png)
![image](https://user-images.githubusercontent.com/64238570/236639848-3efc2f89-c681-4b57-b08b-0d2cf0296e34.png)
![image](https://user-images.githubusercontent.com/64238570/236639890-98e540fa-680b-415a-ab0d-e44873f96ea4.png)

Reply:
God bless you flexbox